hub/core/meta/index_meta.py

```diff
@@ -6,17 +6,23 @@
 
 
 def _create_entry(
-    chunk_names: List[str], start_byte: int, end_byte: int, shape: Tuple[int] = None
+    chunk_names: List[str],
+    start_byte: int,
+    end_byte: int,
+    shape: Tuple[int],
+    dtype: str,
+    compression: str,
 ) -> dict:
     # TODO: replace with `SampleMeta` class
 
     entry = {
         "chunk_names": chunk_names,
         "start_byte": start_byte,
         "end_byte": end_byte,
+        "compression": compression,
```

Comment:
I wouldn't store this data per sample, would be much more preferable be to be per tensor or at least per chunk.

Reply:
tensor_meta contains global tensor information. we are taking in pre-compressed samples as is. this means each sample may have different compressions.